code/game/machinery/autolathe.dm

```diff
@@ -113,20 +116,29 @@
 				categories_associative[cat] = list()
 
 			//Calculate cost
+			var/price = 0	
 			var/list/material_cost = list()
+			
 			for(var/material_id in D.materials)
-				material_cost += list(list(
-					"name" = material_id,
-					"amount" = D.materials[material_id] / MINERAL_MATERIAL_AMOUNT,
-				))
+				if(material_id != "rigid material")
+					price += ore_values["[material_id]"] * D.materials[material_id] / price_factor // also multiplied by 2000, since there are 2000 mat units in a sheet
```

Comment:
```suggestion
					price += round(ore_values["[material_id]"] * D.materials[material_id] / price_factor) // also multiplied by 2000, since there are 2000 mat units in a sheet
```